addons/mail/static/src/components/message/message.js

```diff
@@ -470,6 +471,14 @@ class Message extends Component {
             });
         }
         this._wasSelected = this.isSelected;
+        if (
+            this.message.originThread &&
+            this.threadView &&
+            this.threadView.thread &&
+            this.threadView.thread.model === 'mail.box'
+        ) {
+            this.message.originThread.refreshFollowers();
+        }
```

Comment:
This should not be done in component, especially not after every update.

Reply:
The best would be to return it with `message_format` in the same RPC (`message_fetch`), or in the same bus payload (in case of needaction being sent on bus).